test/openssl/test_engine.rb

```diff
@@ -59,12 +59,21 @@ def test_openssl_engine_cipher_rc4
     end
 
     with_openssl(<<-'end;', ignore_stderr: true)
-      engine = get_engine
+      engine = OpenSSL::Engine.by_id("openssl")
       algo = "RC4"
       data = "a" * 1000
       key = OpenSSL::Random.random_bytes(16)
-      encrypted = crypt_data(data, key, :encrypt) { engine.cipher(algo) }
-      decrypted = crypt_data(encrypted, key, :decrypt) { OpenSSL::Cipher.new(algo) }
+
+      cipher = engine.cipher(algo)
+      cipher.send :encrypt
```

Comment:
Is the `.send` necessary? (is it a private method?)
(same below)

Reply:
Good catch. Fixed and squashed the commits.